properties.js

```diff
@@ -58,6 +57,8 @@ module.exports = {
   // Allowed values: de, en
   language: 'de',
 
+  darkModeActiveProperty: true,
```

Comment:
I'd set the default to false - most projects don't specify dark-mode
```suggestion
  darkModeActiveProperty: false,
```